Modules/_io/_iomodule.h

```diff
@@ -183,4 +185,4 @@ extern PyObject *_PyIO_str_write;
 extern PyObject *_PyIO_empty_str;
 extern PyObject *_PyIO_empty_bytes;
 
-extern PyTypeObject _PyBytesIOBuffer_Type;
+extern Py_EXPORTED_SYMBOL PyTypeObject _PyBytesIOBuffer_Type;
```

Comment:
Why do you export these _io types, they should be private, no?

Reply:
Even if I remove `Py_EXPORTED_SYMBOL` here in the header, it's exported via the definition in `/Modules/_io/bytesio.c`. If I remove it there too, the `_testcapi` module fails to build:
```
running build
running build_ext
INFO: Can't locate Tcl/Tk libs and/or headers
*** WARNING: renaming "_testcapi" since importing it failed: build/lib.linux-x86_64-3.9/_testcapi.cpython-39-x86_64-linux-gnu.so: undefined symbol: _PyBytesIOBuffer_Type
```